fsspec/implementations/reference.py

```diff
@@ -159,7 +159,7 @@ def open_refs(field, record):
             path = self.url.format(field=field, record=record)
             data = io.BytesIO(self.fs.cat_file(path))
             df = self.pd.read_parquet(data, engine="fastparquet")
-            refs = {c: df[c].values for c in df.columns}
+            refs = {c: df[c].to_numpy() for c in df.columns}
```

Comment:
I'm not sure that these are equivalent

Reply:
The documentation of [DataFrame.values](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.values.html#pandas.DataFrame.values) reads:
> Return a Numpy representation of the DataFrame.
> 
> #### ⚠️ Warning
> 
> We recommend using [DataFrame.to_numpy()](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.to_numpy.html#pandas.DataFrame.to_numpy) instead.

Reply:
This is Series.values ( https://pandas.pydata.org/docs/reference/api/pandas.Series.values.html ), which suggests `.array` is the more direct successor. By default, `.to_numpy` has `copy=False`, so maybe it amounts to the same thing. At least we should put in that kwarg explicitly to prevent surprises.